src/vivarium_nih_us_cvd/components/interventions.py

```diff
@@ -35,7 +35,7 @@ def _get_configuration_defaults(self) -> Dict[str, Dict]:
         return {self.configuration_key: LinearScaleUp.configuration_defaults["treatment"]}
 
     def _get_scale_up_dates(self, builder: Builder) -> Tuple[datetime, datetime]:
-        scale_up_config = builder.configuration[self.configuration_key]["date"]
+        scale_up_config = builder.configuration.intervention[self.configuration_key]["date"]
```

Comment:
This looks to the configuration.intervention key just like the super class (LinearScaleUp) does for determining whether the scenario is "baseline" or not